pkg/operation/botanist/component/resourcemanager/resource_manager_test.go

```diff
@@ -458,7 +456,9 @@ var _ = Describe("ResourceManager", func() {
 				}
 			} else {
 				config.Webhooks.CRDDeletionProtection.Enabled = true
+				config.Webhooks.EndpointSliceHints.Enabled = true
 				config.Webhooks.ExtensionValidation.Enabled = true
+				config.Webhooks.SeccompProfile.Enabled = true
```

Comment:
Now, I am just confused. Why is `SeccompProfile` enabled now?

Reply:
See https://github.com/gardener/gardener/pull/7191/files#r1094431856